docs/StardustDocs/topics/ColumnSelectors.md

```diff
@@ -127,42 +94,39 @@ df.select { name.allDfs() }
 ```
 
 </tab>
-<tab title="KProperties">
+<tab title="Strings">
 
 ```kotlin
 // by column name
-df.select { it[Person::name] }
-df.select { (Person::name)() }
-df.select { col(Person::name) }
+df.select { it["name"] }
 
 // by column path
-df.select { it[Person::name][Name::firstName] }
-df.select { Person::name[Name::firstName] }
+df.select { it["name"]["firstName"] }
+df.select { "name"["firstName"] }
```

Comment:
is it really works this way, call [] on String? I don't know about it

Reply:
It is! It's an overloaded "get" operator.